pkg/util/mapper/mapper.go

```diff
@@ -530,6 +530,8 @@ func durationString(duration time.Duration) string {
 }
 
 func parseDuration(duration string) (time.Duration, error) {
+	// Trim prefix of Prometheus format duration
+	duration = strings.TrimPrefix(duration, "offset ")
```

Comment:
Should leading whitespace be trimmed before trimming the prefix? I have no idea, not sure how this is used.